.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: YB4iUiX9WHTj4g24kZnb6o5IcUNg3R9jh
```

Comment:
I don't think we want this publicly available. Can we put in private_config, or otherwise?